libmate-panel-applet/mate-panel-applet.c

```diff
@@ -1820,35 +1925,38 @@ mate_panel_applet_constructor (GType                  type,
 {
 	GObject     *object;
 	MatePanelApplet *applet;
+	MatePanelAppletPrivate *priv;
 
 	object = G_OBJECT_CLASS (mate_panel_applet_parent_class)->constructor (type,
-	                                                                  n_construct_properties,
-	                                                                  construct_properties);
+                                                                               n_construct_properties,
+                                                                               construct_properties);
```

Comment:
Alignment.

Reply:
These lines have not been modified, so I'm going undo the change

Reply:
done